BackEnd/src/main/java/com/javaschool/service/QueueSenderService.java

```diff
@@ -0,0 +1,7 @@
+package com.javaschool.service;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+
+public interface QueueSenderService {
+    String SendRequest(String packageType) throws JsonProcessingException;
```

Comment:
Method naming conventions are `camelCase`
Please change `SendRequest` to `sendRequest` 🤣